golem-rib/src/interpreter/interpreter_stack_value.rs

```diff
@@ -205,7 +205,7 @@ mod internal {
     #[cfg(feature = "json_in_errors")]
     pub fn unable_to_complete_math_operation(left: &ValueAndType, right: &ValueAndType) -> String {
         format!(
-            "Unable to complete math operation for {}, {}",
+            "Unable to complete math operation for operands {}, {}",
```

Comment:
Usually this code shouldn't be hitting if the compiler was strong enough. And this is fixed in this PR.